manuscripts.json

```diff
@@ -96,7 +96,7 @@
         ]
       },
       "msas": ["Neuroscience"],
-      "pdfUrl": "https://www.biorxiv.org/content/10.1101/2022.11.08.515698v2.full.pdf"
+      "pdfUrl": "https://github.com/elifesciences/enhanced-preprints-data/blob/master/data/10.1101/2022.11.08.515698/2022.11.08.515698.pdf"
```

Comment:
Not sure this is correct.

This just points to the pdf in the context of the github repo.

Think it should be: https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/10.1101/2022.11.08.515698/2022.11.08.515698.pdf

Reply:
Haha! Yes, you're right, despite me looking several times 🤦 

Corrected now